clone

```diff
@@ -63,4 +63,7 @@ projectRoot=$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pw
   test -d ffc-sfd-file-processor\
     && >&2 echo "The 'ffc-sfd-file-processor' repository already exists. Skipping." \
     || git clone https://github.com/DEFRA/ffc-sfd-file-processor.git
+  test -d ffc-sfd-file-processor\
```

Comment:
There's duplication here, I assume this line should read `ffc-sfd-file-receiver`?

Reply:
good spot!